src/plots/font_attributes.js

```diff
@@ -86,6 +86,42 @@ module.exports = function(opts) {
             ].join(' ')
         },
 
+        capitalize: opts.noFontCapitalize ? undefined : {
+            editType: editType,
+            valType: 'enumerated',
+            values: ['none', 'word', 'uppercase', 'lowercase'],
```

Comment:
I might suggest
- `'normal'` instead of `'none'`, since `'none'` could be interpreted as no caps, ie lowercase.
- `'upper'` and `'lower'` instead of `'uppercase'` and `'lowercase'` since the name `capitalize` already says we're talking about case.

Reply:
Building on this -- why not just call the attribute `case`?

Then you could have `case: 'upper'`, `case: 'lower'`, and `case: 'title'` and `case: 'normal'`.

Regarding the last one, there's probably a better name for "just leave the casing as-is, don't change it" but can't think of one at the moment.

Reply:
I do like `case` better than `caps`. 
In fact we do have a `caps` attribute for `isosurface` and `volume` which is something different. 
How about `case: 'caps'` instead of`case: 'title'`?

Reply:
Alright. I revisited the API in regards to your comments and other considerations.
For the attribute name `textcase` is declared instead of `capitalize`.
Instead of *word* option I wanted to use *headline* (or *title*) but then I thought we should not use them as `text-transform: 'capitalize'` is different in a way that it capitalizes all the words. See `capitalize` notes [here](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform). As a results *caps* is used in the API.

Reply:
I'm fine with `textcase` but I don't like `caps` -- I would expect that to mean all-uppercase. Maybe `wordcaps`? I don't love that either but I think it's clearer than `caps`.